### ISSUE TITLE
Compatible with tmux 2.2

### ISSUE THREAD

**Issue by bookletchoir** [powerline/powerline#1574]
tmux 2.2 is now detect utf8 automatically, hence the option `set-utf8 on` no longer works.

And, powerline for tmux crashed when trying to call that option from tmux config.

```bash
➜  ~  powerline-config tmux setup
zsh: correct 'tmux' to '.tmux' [nyae]? n
unknown option: status-utf8
Traceback (most recent call last):
  File "/usr/bin/powerline-config", line 22, in <module>
    args.function(pl, args)
  File "/usr/lib/python3.5/site-packages/powerline/commands/config.py", line 15, in __call__
    self.function(*args, **kwargs)
  File "/usr/lib/python3.5/site-packages/powerline/bindings/config.py", line 212, in tmux_setup
    source_tmux_files(pl, args, tmux_version=tmux_version, source_tmux_file=stf)
  File "/usr/lib/python3.5/site-packages/powerline/bindings/config.py", line 74, in source_tmux_files
    source_tmux_file(os.path.join(TMUX_CONFIG_DIRECTORY, 'powerline-base.conf'))
  File "/usr/lib/python3.5/site-packages/powerline/bindings/config.py", line 199, in source_tmux_file_nosource
    run_tmux_command(*args)
  File "/usr/lib/python3.5/site-packages/powerline/bindings/tmux/__init__.py", line 32, in run_tmux_command
    _run_tmux(subprocess.check_call, args)
  File "/usr/lib/python3.5/site-packages/powerline/bindings/tmux/__init__.py", line 27, in _run_tmux
    return runner([get_tmux_executable_name()] + list(args))
  File "/usr/lib/python3.5/subprocess.py", line 584, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['tmux', 'set', '-g', 'status-utf8', 'on']' returned non-zero exit status 1
```

```bash
➜  ~  tmux set -g status-utf8 on
unknown option: status-utf8
```

Full changelog of tmux: https://raw.githubusercontent.com/tmux/tmux/master/CHANGES
I'm using powerline 2.3 from stable release.

**Comment by ZyX-I** [comment]
Do use search.

**Comment by aiguofer** [comment]
Wow... this is why some people never contribute to open source.

You could at least put a link to the duplicate issue if you're going to mark it as so. Also, generally speaking a bug isn't fixed until the commit is in a release since most people only installed versioned packages; if you had left the original bug open until it was in a release then others would more easily find this since github defaults to only searching open bugs.

**Comment by ZyX-I** [comment]
This is now in release.

**Comment by ZyX-I** [comment]
And bug cannot stay open until fix is in release because this is impossible to do if you want to use `Fixes #123` in commit message. I have never seen projects here that leave bugs open like this.

**Comment by pizzaops** [comment]
I agree that the bug shouldn't remain open, especially if it's a duplicate anyway, but marking it a dupe and then saying "do use search" instead of just linking to the ticket you've already found is pretty user-hostile behavior...kind of a bummer to see this.
